test/utils.test.js

```diff
@@ -0,0 +1,65 @@
+import { setFilterOnly } from '../src/utils'
+import { schemaToCSVTransforms } from '../src/schemaToCSVTransforms'
+
+describe('utils', () => {
+  it('setFilterOnly', async () => {
+    let tree = {
+      key: 'root',
+      children: [
+        { key: 'a', type: 'facet', field: 'a' },
+        { key: 'b', type: 'facet', field: 'b' },
+        { key: 'results', type: 'results' },
+      ],
+    }
+    let filtered = setFilterOnly(tree)
+    expect(filtered).toEqual({
+      filterOnly: true,
+      key: 'root',
+      children: [
+        { filterOnly: true, key: 'a', type: 'facet', field: 'a' },
+        { filterOnly: true, key: 'b', type: 'facet', field: 'b' },
+        { filterOnly: true, key: 'results', type: 'results' },
+      ],
+    })
+  })
+  it('schemaTransforms', () => {
```

Comment:
Generally I would match the test file with the file being tested and not combine multiple files like this, it starts to get really confusing as to what's being tested as the testing surface grows.  Also this seems to be missing tests for other util functions.